torch/utils/data/datapipes/datapipe.py

```diff
@@ -132,6 +132,14 @@ def set_reduce_ex_hook(cls, hook_fn):
             raise Exception("Attempt to override existing reduce_ex_hook")
         IterDataPipe.reduce_ex_hook = hook_fn
 
+    def __repr__(self):
+        # Instead of showing <torch. ... .MapperIterDataPipe object at 0x.....>, return the class name
+        return str(self.__class__.__qualname__)
+
+    def __str__(self):
+        # Instead of showing <torch. ... .MapperIterDataPipe object at 0x.....>, return the class name
+        return str(self.__class__.__qualname__)
+
```

Comment:
What's the difference between `__str__` and `__repr__`?
And, can we add a registration stub in it? Then, when we work on the graph visualization or buffer/spec visualization, we will be able to override these functions.

Reply:
Certain built-in functions choose to call `__repr__` (e.g. when exception is raised, `__repr__(obj)` is called). 

What do you mean by registration stub? Something similar to the `set_state_hook`?

Reply:
The stubs have been added!